coretex/cli/config.py

```diff
@@ -0,0 +1,151 @@
+from typing import Optional, Tuple
+from pathlib import Path
+
+from tabulate import tabulate
+
+import click
+
+from .utils import arrowPrompt
+from ..networking import networkManager
+from ..statistics import getAvailableRamMemory
+from ..configuration import loadConfig, saveConfig, isUserConfigured, isNodeConfigured
+
+
+def authenticate(retryCount: int = 0) -> Tuple[str, str, str, str]:
+    if retryCount >= 3:
+        raise click.ClickException("") #raise errror
+
+    username = click.prompt("Email", type = str)
+    password = click.prompt("Password", type = str, hide_input = True)
+
+    response = networkManager.authenticate(username, password, False)
+
+    if response.hasFailed():
+        click.echo("Failed to authenticate. Please try again...")
+        return authenticate(retryCount + 1)
+
+    jsonResponse = response.getJson(dict)
+
+    return username, password, jsonResponse["token"], jsonResponse["refreshToken"]
+
+
+def registerNode(name: str) -> Optional[str]:
+    params = {
+        "machine_name": name
+    }
+    response = networkManager.post('service', params = params)
+
+    if response.hasFailed():
+        click.echo("Failed to configure node. Please try again...")
+        return None
+
+    return str(response.getJson(dict)["access_token"])
```

Comment:
`access_token` is already a string, why are you creating a new string object here?

Reply:
Regarding python response.getJson(dict)["access_token"] is type of Any

Reply:
That might be what the linter is "guessing", but the actual `type(response.getJson(dict)["access_token"])` will be `str`. Perform validation instead of creating a new object here